### PR TITLE
[docs] develop branch - update links and list of tools as a bullet list 

### LINKED ISSUE #861: corrections for installation page
at the bottom on this page
https://developers.eos.io/manuals/eosio.cdt/latest/installation
there's a couple of things wrong

1. the list of tools "eosio-abidiff eosio-ranlib eosio-ar eosio-objdump eosio-readelf" should be rendered as a bullet list
2. the "see here" link is wrong (https://developers.eos.io/manuals/eosio.cdt/latest/installation/#guided_installation_or_building_from_scratch) it should read the following: https://developers.eos.io/manuals/eosio.cdt/latest/installation/#guided-installation-or-building-from-scratch

don't forget to fix it in development and release branch

### PR DESCRIPTION
**Pull request by iamveritas**
update links and list of tools as a bullet list
resolves #861
sister PR #1031
sister PR #1032

### CODE REVIEW

File: docs/02_installation.md [hunk 1]
```diff
@@ -78,13 +85,13 @@ $ sudo rm /usr/local/bin/eosio-*
 * eosio-objdump
 * eosio-readelf
 
-Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided_installation_or_building_from_scratch)
-eosio-abidiff
-eosio-ranlib
-eosio-ar
-eosio-objdump
-eosio-readelf
+The following tools are not installed after brew install, you get them when build the repository and install it from scratch, [see here](#guided-installation-or-building-from-scratch):
```
Comment (bobgt) [inline docs/02_installation.md:88]: Tiny suggestion:

The following tools are not installed after brew install; you get them when you build the repository and install it from scratch, [see here](#guided-installation-or-building-from-scratch):

Comment (iamveritas) [reply docs/02_installation.md]: thank you
updated

File: docs/02_installation.md [hunk 2]
```diff
@@ -7,39 +7,46 @@ EOSIO.CDT currently supports Mac OS X brew, Linux x86_64 Debian packages, and Li
 **If you have previously installed EOSIO.CDT, run the `uninstall` script (it is in the directory where you cloned EOSIO.CDT) before downloading and using the binary releases.**
 
 ## Mac OS X Brew Install
+
 ```sh
 $ brew tap eosio/eosio.cdt
```
Comment (lparisc) [inline docs/02_installation.md:12]: consider removing the shell prompt $ from all these shell commands

**Review by lparisc — APPROVED** [review]

**Review by bobgt — APPROVED** [review]

File: docs/02_installation.md (hunk 2 above)
Comment (iamveritas) [reply docs/02_installation.md]: done

**Pull request merged**